> @WalterKolczynski-NOAA , the linter failure is
> 
> ```
> ✋ Defects, NEEDS INSPECTION
> Error: SHELLCHECK_WARNING:
> ./jobs/JGLOBAL_ATM_ANALYSIS_FINALIZE:11:28: note[SC2153]: Possible misspelling: PDY may not be assigned. Did you mean gPDY?
> 
> Error: SHELLCHECK_WARNING:
> ./jobs/JGLOBAL_ATM_ANALYSIS_INITIALIZE:10:28: note[SC2153]: Possible misspelling: PDY may not be assigned. Did you mean gPDY?
> ```
> 
> For both jobs the offending line is
> 
> ` GDATE=$(date +%Y%m%d%H -d "${PDY} ${cyc} - ${assim_freq} hours")`
> 
> Both the initialize and finalize jobs use files from the previous cycle. The rocoto workflow defines `PDY`.
> 
> What is the preferred solution to resolve the linter failure?

place a line like this before:
```
# shellcheck disable=SC2153
```

--- a/jobs/JGLOBAL_ATM_ANALYSIS_FINALIZE
+++ b/jobs/JGLOBAL_ATM_ANALYSIS_FINALIZE
@@ -8,9 +8,10 @@ source "${HOMEgfs}/ush/jjob_header.sh" -e "atmanlfinal" -c "base atmanl atmanlfi
 ##############################################
 # Set variables used in the script
 ##############################################
+# shellcheck disable=SC2153
 GDATE=$(date +%Y%m%d%H -d "${PDY} ${cyc} - ${assim_freq} hours")
-export gPDY=${GDATE:0:8}
-export gcyc=${GDATE:8:2}
+gPDY=${GDATE:0:8}
+gcyc=${GDATE:8:2}
 GDUMP="gdas"
 
 

--- a/jobs/JGLOBAL_ATM_ANALYSIS_INITIALIZE
+++ b/jobs/JGLOBAL_ATM_ANALYSIS_INITIALIZE
@@ -7,11 +7,12 @@ source "${HOMEgfs}/ush/jjob_header.sh" -e "atmanlinit" -c "base atmanl atmanlini
 ##############################################
 # Set variables used in the script
 ##############################################
+# shellcheck disable=SC2153
 GDATE=$(date +%Y%m%d%H -d "${PDY} ${cyc} - ${assim_freq} hours")
-export gPDY=${GDATE:0:8}
-export gcyc=${GDATE:8:2}
+gPDY=${GDATE:0:8}
+gcyc=${GDATE:8:2}
 GDUMP="gdas"
-export GDUMP_ENS="enkf${GDUMP}"
+GDUMP_ENS="enkf${GDUMP}"
 
 
 ##############################################

